ext/spl/spl_array.stub.php

```diff
@@ -5,15 +5,13 @@
 class ArrayObject implements IteratorAggregate, ArrayAccess, Serializable, Countable
 {
     /**
-     * @var int
      * @cvalue SPL_ARRAY_STD_PROP_LIST
```

Comment:
I'd love if these PHPDoc blocks were in one line:
```suggestion
     /** @cvalue SPL_ARRAY_STD_PROP_LIST */
```

Reply:
Changed all of the PHPDoc blocks to one-line. The same done for other PRs.